### PR TITLE
Lizard food: adds burger variants and a few extras

### PR DESCRIPTION
**Pull request by xXPawnStarrXx**
## About The Pull Request
This PR is inspired by [SS13 TV](https://youtu.be/gVXki-1uBLY?si=l9QZyVFW1CyzIEL_&t=73) [The best show on air.](https://youtu.be/gVXki-1uBLY?si=hrQ8NADTu0WwmH7n&t=223)
I wanted to help port the great human cuisine over to be something lizards could comprehend.
I'm intending to add: 7 Burger variants (Rat, Plain, Chicken, Fish, Sssloppy Moe, Rib and RootGuffin)
image
I also am adding two extra foods: Korta Wellington and [Meatsicles.](https://media.tenor.com/CGDqgg8bb9AAAAAC/meat-popsicle-5th-element.gif)
image
I'm not done yet! For human chefs, rejoice! I'm adding a way to make vinegar from grape juice and enzyme (removing the wine recipe because that just becomes grappa instantly.) and the ability to 'fold' your pickle jars back into beakers to let you recycle the jar to make more pickles without having to trek to the nearby lathe or autolathe.

Updated meatsicles. 
image

## Why It's Good For The Game
More food for lizards means a chef doesn't need to harass botany for more and more plants just to satisfy the growing lizard population. You can feed humans the same burgers they love without aggression from the ~~subhumans~~ lizardfolk who are hungry too.
Vinegar is now craftable, which means if your grapes and enzyme is of high enough quality, it'll not impact your food quality. However, that requires a competant botanist and more work. You can choose between cargo or saving like 20cr for your work.
Not being able to just refill an empty jar didn't make any sense to me, I've made pickles IRL and have never encountered that.
## Changelog
:cl:
add: Added new lizard variants of existing foods for equality of edibility.
qol: made pickle jars reusable and vinegar craftable.
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/frozen.dm [hunk 1]
```diff
@@ -434,3 +434,16 @@
 	foodtypes = DAIRY | SUGAR
 	venue_value = FOOD_PRICE_NORMAL
 	crafting_complexity = FOOD_COMPLEXITY_3
+
+/obj/item/food/popsicle/meatsicle
+	name = "Meatsicle"
+	desc = "A horrific abomination of raw meat, glazed with sugar on a stick, then frozen."
```
Comment (nikothedude) [inline code/game/objects/items/food/frozen.dm:440]: this is amazing

Comment (xXPawnStarrXx) [reply code/game/objects/items/food/frozen.dm:440]: https://media.tenor.com/CGDqgg8bb9AAAAAC/meat-popsicle-5th-element.gif

**Comment by ZephyrTFA** [comment]
species change, without prior approval?
It's more likely than you think

**Comment by krookodilehunter** [comment]
can you make meatsicles from different types of meat? IE xeno, carp, chicken, etc? i don't wanna make more work for you but i think different sprites for different meatsicles would be cute!

**Comment by xXPawnStarrXx** [comment]
> can you make meatsicles from different types of meat? IE xeno, carp, chicken, etc? i don't wanna make more work for you but i think different sprites for different meatsicles would be cute!

I don't see why not. I could add them later or if the PR is merged do an additional one with extra variants. Uncooked chicken seems like a bad idea though.

**Comment by nikothedude** [comment]
its a great idea as long as it gives salmonella

**Review by Fikou — CHANGES_REQUESTED** [review]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by xXPawnStarrXx** [comment]
> Your current title and or body content does not meet requirements, so please update it.
I previously had one set, the page refreshed so it defaulted to the branch name. Fixed it.

**Comment by carshalash** [comment]
> > can you make meatsicles from different types of meat? IE xeno, carp, chicken, etc? i don't wanna make more work for you but i think different sprites for different meatsicles would be cute!
> 
> I don't see why not. I could add them later or if the PR is merged do an additional one with extra variants. Uncooked chicken seems like a bad idea though.

https://matadornetwork.com/read/raw-chicken-sashimi-tokyo/ 
he doesn't know...

**Review by Wallemations — CHANGES_REQUESTED** [review]
The only thing I'm not fond of with the sprites is the meatsicle's perspective
If it's gonna be vertical I would lay it flat on the table to conform with 3/4ths

File: code/game/objects/items/food/lizard.dm [hunk 2]
```diff
@@ -839,3 +839,109 @@
...
+	name = "rootrib"
+	desc = "An elusive rib shaped burger with limited availablity across the galaxy. Now meeting subhuman requirements."
+	icon_state = "rootrib"
+	icon = 'icons/obj/food/lizard.dmi'
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 2,
+		/datum/reagent/consumable/nutriment/protein = 7,
+		/datum/reagent/consumable/nutriment/vitamin = 4,
+		/datum/reagent/consumable/bbqsauce = 1,
+		)
```
Comment (MrMelbert) [inline code/game/objects/items/food/lizard.dm:896]:
```suggestion
	)
```

**Comment by xXPawnStarrXx** [comment]
> The only thing I'm not fond of with the sprites is the meatsicle's perspective If it's gonna be vertical I would lay it flat on the table to conform with 3/4ths

image
Updated with extra crackhead energy.

**Comment by nikothedude** [comment]
thats a shame, i liked the old meatsicle way more, but if thats what it takes to get it in oh well ig

**Comment by Wallemations** [comment]
I mean I liked the old shape but sure that at least fits the perspective better

**Comment by xXPawnStarrXx** [comment]
> I mean I liked the old shape but sure that at least fits the perspective better

I took seven attempts at getting the perspective better and hated every one. So I went with the crackhead inside.

**Comment by Loafin34** [comment]
can you make us some other way for bootleg wine? any way besides booze dispenser is a good way. but great job!!

**Comment by xXPawnStarrXx** [comment]
> can you make us some other way for bootleg wine? any way besides booze dispenser is a good way. but great job!!

Grapes in a fermenting barrel.

**Comment by krookodilehunter** [comment]
what about something like this for the meatsicles? 
popsicleflipped

**Review by Ghommie — APPROVED** [review]
The suggested meatsicle sprites with the vertically flipped meat slab looks better though.

**Comment by xXPawnStarrXx** [comment]
> The suggested meatsicle sprites with the vertically flipped meat slab looks better though.

It does, I have a habit of breaking more when I touch stuff, so I'm going to let them resprite it should they choose, but I don't trust my Jayson Gibson skulled ass.

**Comment by Ghommie** [comment]
> > The suggested meatsicle sprites with the vertically flipped meat slab looks better though.
> 
> It does, I have a habit of breaking more when I touch stuff, so I'm going to let them resprite it should they choose, but I don't trust my Jayson Gibson skulled ass.

It reads a speck like garnished lazyness to me, given you've already done the sprites for it. It isn't like you're a total stranger to it... but ok...

**Comment by xXPawnStarrXx** [comment]
> It reads a speck like garnished lazyness to me, given you've already done the sprites for it. It isn't like you're a total stranger to it... but ok...
I managed to screw up two PRs somehow, to the point where I literally closed them because I don't know how.

**Comment by krookodilehunter** [comment]
i went ahead and made a full set of meatsicle sprites, if that's okay! i just really like the idea :) 
meatsicles
i also have the individual sprites for each to be layered over a popsicle stick if you'd like them

**Comment by nikothedude** [comment]
the point of using the current sprites was to comply with the wallening's perspective, that doesnt do that

**Comment by Ghommie** [comment]
Oh, oh. Now I see why the new meatsicles sprites are like that. Had I know this earlier (read Wallemations' comment), I wouldn't have called it out as lazyness, my bad. Though I don't particularly agree with Wallem's view on it (the perspective is the same as the wellington to be fair), it feels like splitting hair for me.

**Comment by krookodilehunter** [comment]
oh shoot, yeah you're right

**Comment by krookodilehunter** [comment]
tried my hand at changing the perspective. is this any better?
meatsicles
(also i hope i'm not over-stepping or anything)

**Pull request merged**